javascript/builtins/intl/NumberFormat.json

```diff
@@ -859,44 +859,6 @@
                 "experimental": false,
                 "standard_track": true,
                 "deprecated": false
-              },
```

Comment:
@queengooborg I removed this as there was an error 
- `Error → ADDTIONAL PROPERTY must NOT have additional properties"`
- `result_useGrouping_property is not expected to be here!`
- ` Error → Property sorting error on undefined`
But I still want to have something here:

Essentially three is a useGrouping property that might be specified in the object returned by resolvedOptions. This has changed. This feature was supposed to indicate how.

What do I need to do?

Reply:
I don't see a removal in the changes so I'm assuming this was reverted?

Reply:
Fixed and added back.